torntools/changelog.js

```diff
@@ -1,6 +1,6 @@
 export default {
 	"v5.7 - xxxxxx, xx. 2021": {
-		Features: ["Add option to export War Report as CSV. - Sashank999"],
+		Features: ["Add option to export War Report as CSV. - Sashank999", "Add 'I'm Chaining' button to stop warning for 30 minutes on Attack Page. - Sashank999"],
```

Comment:
It's not really a feature, I'd think. It's a change to an existing feature.

Reply:
Changed.